main.css

```diff
@@ -0,0 +1,573 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: white;
+  font-family: 'Lato', sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  overflow-x: hidden;
+}
+
+.hide {
+  display: none;
+}
+
+@media screen and (min-width: 375px) and (max-width: 767px) {
+  header {
+    width: 100%;
+    height: 100%;
+    padding: 15px;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  .mobile-menu i {
+    margin-bottom: 60px;
+  }
+
+  .menu {
+    background-color: #ec5242;
+    color: #fff;
+    padding-top: 100px;
+    font-size: 30px;
+    position: absolute;
+    width: 100%;
+    height: 800px;
+    mix-blend-mode: multiply;
+    top: -2%;
+    left: -2%;
+    right: -2%;
+    backdrop-filter: blur(15px);
+  }
+
+  .menu ul {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-end;
+    text-align: left;
+  }
+
+  .menu ul a {
+    text-align: left;
+    color: white;
+  }
+
+  .menuBtn {
+    width: 20px;
+  }
+
+  .close-menu {
+    position: absolute;
+    top: 0;
+    right: 0;
+    padding: 30px;
+    color: #fff;
+    z-index: inherit;
+  }
+
+  body::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    opacity: 0.1;
+    z-index: -1;
+    background-image: url('./img/back-img1.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .mobile-menu {
+    display: flex;
+    margin-top: 10px;
+  }
+
+  .desk_nav {
+    display: none;
+  }
+
+  .description {
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+    margin-left: 50px;
+  }
+
+  .desc-header {
+    font-size: 25px;
+    color: #ec5242;
+  }
+
+  .description h1 {
+    font-size: 250%;
+    font-weight: 700;
+    color: #ec5242;
+    line-height: 38px;
+  }
+
+  .p-1 {
+    width: 80%;
+    background-color: #dddddded;
+    border: 3px solid white;
+    padding: 5px;
+    margin: 5px 5px;
+  }
+
+  .description .time {
+    margin-top: 20px;
+    font-weight: 900;
+  }
+
+  .venue {
+    font-weight: 600;
+    margin: 7px 10px 200px;
+  }
+
+  .program {
+    width: 100%;
+    height: 800px;
+    background-image: url('./img/pro-mob-back.jpg');
+    background-color: #1e2023;
+    color: #fff;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 50px;
+  }
+
+  .cards {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(5, 1fr);
+    align-items: center;
+    justify-content: center;
+    gap: 10px;
+    width: 100%;
+    padding-top: 50px;
+    margin-left: 5px;
+  }
+
+  .card {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+    background-color: #383b41;
+    width: 100%;
+    height: 100px;
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 make your cards responsive for a better UI to be similar to the original design.

| Original design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/234665874-2d418639-dfce-44ca-9446-d4f99b7f9950.png) | ![image](https://user-images.githubusercontent.com/58642949/234666024-19baac56-a2cc-4244-8250-7530864fb35f.png) |